src/main/java/com/cburch/logisim/fpga/prefs/SoftwaresOptions.java

```diff
@@ -214,12 +222,36 @@ public SoftwaresOptions(PreferencesFrame window) {
     c.weightx = 1.0;
     c.fill = GridBagConstraints.HORIZONTAL;
     add(vivadoPathField, c);
-
     c.gridx = 2;
     c.gridy = gridY++;
     c.fill = GridBagConstraints.NONE;
     add(vivadoPathButton, c);
 
+    c.gridx = 0;
```

Comment:
We try to avoid single character names. Is there a better name than `c`? I realize it is an existing variable so an appropriate response may be that it is beyond the scope of this PR.